packages/server-wallet/src/config/defaults.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-process-env */
+import {
+  LoggingConfiguration,
+  NetworkConfiguration,
+  OptionalDatabaseConfiguration,
+  OptionalServerWalletConfig,
+  ServerWalletConfig,
+} from './types';
+
+export const defaultDatabaseConfiguration: OptionalDatabaseConfiguration & {
+  connection: {host: string; port: number};
+} = {
+  pool: undefined,
+  debug: false,
+  connection: {host: 'localhost', port: 5432, user: 'postgres'},
+};
+
+/**
+ * Network config
+ */
+
+export const defaultNetworkConfiguration: NetworkConfiguration = {
+  chainNetworkID: '0x00',
+  erc20Address: undefined,
+  ethAssetHolderAddress: undefined,
+  erc20AssetHolderAddress: undefined,
+};
+
+/**
+ * Logging and Metrics config
+ */
+export const defaultLoggingConfiguration: LoggingConfiguration = {
+  logLevel: 'info',
+  logDestination: 'console',
+};
+
+export const defaultMetricConfiguration = {timingMetrics: false};
+
+/**
+ * Server Wallet config
+ */
+
+/**
+ * These are the default values that will be used by the server wallet
+ * if not overidden in the provided config
+ */
+export const defaultConfig: OptionalServerWalletConfig = {
+  databaseConfiguration: defaultDatabaseConfiguration,
```

Comment:
`databaseConfiguration` probably better named `postgresqlConfiguration` as it is quite specific